posts/2025-03-02-Rustup-1.28.0.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "Announcing Rustup 1.28.0"
+author: The Rustup Team
+---
+
+The rustup team is happy to announce the release of rustup version 1.28.0.
+[Rustup][install] is the recommended tool to install [Rust][rust], a programming language that is empowering everyone to build reliable and efficient software.
+
+## What's new in rustup 1.28.0
+
+This new release of rustup has been a long time in the making and comes with substantial changes.
+
+Before digging into the details, it is worth mentioning that [Chris Denton](https://github.com/chrisdenton) has joined the team.
+Chris has a lot of experience contributing to Windows-related parts of the Rust Project, expertise which was sorely lacking in existing team members, so we're happy to have him on board to help address Windows-specific issues.
```

Comment:
```suggestion
Chris has a lot of experience contributing to Windows-related parts of the Rust Project -- expertise we were previously lacking -- so we're happy to have him on board to help address Windows-specific issues.
```

It reads a bit better and more positively to write that aside in the second person than in the passive voice.